big_tests/tests/privacy_SUITE.erl

```diff
@@ -29,22 +29,21 @@
 %%--------------------------------------------------------------------
 
 all() ->
-    [{group, management},
+    [
+     {group, management},
```

Comment:
Minor: no reason to change this, especially that other lists do not contain such extra newlines.